config.yaml

```diff
@@ -114,6 +114,11 @@ params:
     subtitle: Easy to connect us with these open platforms
     row:
     - list:
+      - icon: fa-telegram
+        title: Telegram
+        description: >-
+          Join **PythonPune** telegram group to chat with us.
```

Comment:
```suggestion
          Join **PythonPune** Telegram group to chat with us (linked with the IRC channel).
```
How about this?

Reply:
Looks good. I will make the changes as suggested. :)

Reply:
Updated. Thanks for suggestion.